easybuild/easyblocks/n/nvhpc.py

```diff
@@ -205,6 +216,19 @@ def sanity_check_step(self):
         custom_commands = ["%s -v" % compiler for compiler in compiler_names]
         super(EB_NVHPC, self).sanity_check_step(custom_paths=custom_paths, custom_commands=custom_commands)
 
+        # compile minimal example using -std=c++20 to catch issue where it picks up the wrong GCC
+        # (as long as system gcc is < 9.0)
+        # see: https://github.com/easybuilders/easybuild-easyblocks/pull/3240
+        tmpdir = tempfile.mkdtemp()
+        write_file(os.path.join(tmpdir, 'minimal.cpp'), NVHPC_MINIMAL_EXAMPLE)
+        os.chdir(tmpdir)
+        cmd = "nvc++ -std=c++20 minimal.cpp -o minimal"
```

Comment:
@jfgrimm Let's run this via `custom_commands` instead, since there's a whole bunch of stuff going on here we don't really need, see https://github.com/jfgrimm/easybuild-easyblocks/pull/3

Reply:
merged